processing/src/main/java/io/druid/query/datasourcemetadata/DataSourceQueryQueryToolChest.java

```diff
@@ -120,9 +120,8 @@ public boolean apply(T input)
   public ServiceMetricEvent.Builder makeMetricBuilder(DataSourceMetadataQuery query)
   {
     return new ServiceMetricEvent.Builder()
-        .setUser2(DataSourceUtil.getMetricName(query.getDataSource()))
-        .setUser4(query.getType())
-        .setUser6("false");
```

Comment:
this information is not really needed in the new schema. These queries always have no filters